book/src/proptest-derive/pd/index.md

```diff
@@ -0,0 +1,9 @@
+# The `proptest-derive` crate
```

Comment:
Hmm this file seems to be an unused duplicate?

Reply:
I plan to deduplicate it once I start actually working on the proptest section. Roughly, my plan is to move most of the exposition in `lib.rs` to the book, and then generate the readme from sections of the book.

Reply:
Ah :)